lsp/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "module": "UMD",

```

Comment:
We should be able to start using ES modules (`node16`) because we're bundling node. UMD is kind of dated.